src/tests/store/auth/registrationStore.test.js

```diff
@@ -2,19 +2,23 @@ import { createStore } from 'redux';
 import rootReducer from '../../../reducers/rootReducer';
 import * as registrationActions from '../../../actions/auth/registrationAction';
 
-it('should handle registering users', () => {
-  const store = createStore(rootReducer, {});
-  const responseData = {
-    message: 'successful registration',
-    registered: true,
-    registering: false,
-  };
+// it('should handle registering users', () => {
```

Comment:
@Manorlds-Eaglespark I don't understand the relevance of commenting(deactivating) this test to your feature implementation

Reply:
This was done during testing. No harm was intended. Thanks :-)